<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dispensador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>721</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Usuario</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatura:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Tipos de Bebidas:</string>
      </property>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Coca Cola</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>96</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pepsi</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fanta</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Manaos</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quilmes</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_9">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fernet</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>DRLemon</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_12">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>7UP</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBoxPort">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>DESCONECTAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCONECTAR">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>CONECTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="Imagen2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccione Puero:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Barra de Proceso</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>481</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>20</width>
        <height>291</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>681</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>421</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha y hora:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Estado:    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelEstado">
         <property name="text">
          <string>DESCONECTADO</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Administrador</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_NOMBRE">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PASSWORD">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonINGRESAR">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>INGRESAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SALIR">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SALIR</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>70</y>
        <width>691</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string> Fecha y Hora:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuraciones Avanzadas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GUARDAR">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>GUARDAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMsJ">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>591</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>A1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>A2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C2</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_AGREGAR">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_BORRAR">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>260</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>BORRAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CREAR">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>CREAR</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
